specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/securitydomain.json

```diff
@@ -250,7 +256,7 @@
                   "type": "integer",
                   "minimum": 2,
                   "maximum": 10,
-                  "description": "Number of keys required for security domain"
+                  "description": "The number of keys (minimum 2 and maximum 10) required for security domain. "
```

Comment:
Wasn't the minimum 3 as mentioned above? Or is that different in this case (in which case, shouldn't the maximum be 9 then)?